draft-ietf-quic-tls.md

```diff
@@ -2105,67 +2105,67 @@ The unprotected header includes the connection ID and a 4-byte packet number
 encoding for a packet number of 2:
 
 ~~~
-c3ff000020088394c8f03e5157080000449e00000002
+c300000001088394c8f03e5157080000449e00000002
```

Comment:
Is this implying that implementors of draft-33 should use the `00000001` version?

Reply:
Yes, see #4430.  Though I would (strongly) encourage people not to deploy that version until we get a little further through the process.

Reply:
Can you clarify what "a little further through the process" means? Having folks deploy `00000001` before the RFC is published in its immutable glory could prevent us from making last-minute fixes.

Reply:
Yeah.  I'm just a little scared of what the IESG might uncover.  Certain IESG members have a reputation for being quite thorough.  They probably won't find anything substantive, but I would like to keep the option just a tiny bit available until their reviews are in.

Reply:
To be honest I'm not personally worried about IESG review; I'm more concerned about what we'll find when we actually deploy this protocol in production (no one has launched 0-RTT or migration yet as far as I know).

Reply:
Given that everyone is retaining h3-29, I would suggest that we continue with that until at least after IESG review. There's nothing left in the process at this point that is expected to wait until we have more deployment experience, given that IETF LC is officially complete.

I understand and agree with your concern, @DavidSchinazi, but given that we don't have a specific point in time where we can expect to see migration or 0-RTT deployed, I don't see that waiting is a useful strategy. I would argue that we can leverage v2 for this if we need to fix something.

Reply:
I want final values in the document, because they will need to be tested.  I've already found one problem as a result of implementing this change.  I don't want any surprises.  That said, yes, keep up with the -29 deployments for now.  It's just as good.

And you just reminded me that I need to tweak the ALPN values in the sample.  That's going to suck, but best that I caught it.

Reply:
And I just worked out that the example uses an ALPN of "alpn", so there is no need to update.  Of course, someone could call me on that being a potentially valid value rather than one reserved for examples, but then I wouldn't care.

Reply:
I'm in agreement with you that we need to test a few things before publication, it's just the matter of when we switch to the official codepoints. But I think you did hit the nail on the head that at the end of the day, it's going to be h3-29 for now, then v1 when available - nothing we do between the two will actually matter